content/docs/howto/negate-a-disjunction/en.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Negate a disjunction
+tags:
+- validation
+- language
+- commented cue
+authors:
+- hansbogert
+relatedIssues:
+- https://github.com/cue-lang/cue/discussions/3229
+- https://github.com/cue-lang/docs-and-content/issues/160
+toc_hide: true
+---
+
+This [Commented CUE]({{< relref "docs/howto#commented-cue-guides" >}})
+demonstrates how to negate a disjunction.
+
+{{{with code "en" "example"}}}
+! exec cue vet example.cue
+cmp stderr out
+-- example.cue --
+package example
+
+// Define a disjunction
+#oneTwoThree: "one" | "two" | "three"
+
+// Define a concrete value
+four: "four"
+// define a hidden helper value which makes sure
+// that `four` is not #oneTwoThree by comparing
+// their unification with bottom
+_four_ok: true & ((four & #oneTwoThree) == _|_)
```

Comment:
I'd present the `A` and `B` fields together (uncommented, as they're so trivial), then a blank line, and then a commented block that presents the hidden field for each data field. That way a single comment will suffice for both hidden fields.